source/NVDAObjects/__init__.py

```diff
@@ -328,7 +339,13 @@ def objectFromPoint(x, y):
 		"""
```

Comment:
as you've rewritten this function, could you update this function header to:
- move the type hints from the docstring into the function header
- update the docstring to use sphinx syntax